assets/src/js/admin/admin-export.js

```diff
@@ -67,6 +67,40 @@ jQuery( document ).ready( function( $ ) {
 		} );
 	}
 
+    /**
+     * Show/hide Donor Export 'Search By' option if the date inputs change
+     *
+     * @unreleased
+     */
+    let startDate, endDate;
+    //wait a bit before binding events so datepicker can initialize
```

Comment:
@jonwaldstein No we do not, and this will introduce a race condition. I will see if I can modify the global `datepicker` object  before the datepicker is actually invoked (since options on a specific datepicker will get overwritten once it's initialized).